src/DeferredResult.ts

```diff
@@ -0,0 +1,162 @@
+/**
+ * @module
+ *
+ * The `DeferredResult` type is simply a convenient alias for a {@link Deferred} with an
+ * inner value of a result {@link Result}. The `DeferredResult` type is frequently useful
```

Comment:
Does this `{@link Result}` work without having to reference the module here?

Reply:
@rmjohnson checked and actually it does work, I think because it's imported with the same name below